ci/jenkins/test.sh

```diff
@@ -395,7 +395,7 @@ function deliver_antrea_windows {
         # Some tests need us.gcr.io/k8s-artifacts-prod/e2e-test-images/agnhost:2.13 image but it is not for windows/amd64 10.0.17763
         # Use e2eteam/agnhost:2.13 instead
         harbor_images=("sigwindowstools-kube-proxy:v1.18.0" "agnhost:2.13" "agnhost:2.13" "agnhost:2.13" "agnhost:2.29" "e2eteam-jessie-dnsutils:1.0" "e2eteam-jessie-dnsutils:1.0" "e2eteam-pause:3.2" "e2eteam-pause:3.2" "e2eteam-busybox:1.29-windows-amd64-1809")
-        antrea_images=("sigwindowstools/kube-proxy:v1.18.0" "e2eteam/agnhost:2.13" "us.gcr.io/k8s-artifacts-prod/e2e-test-images/agnhost:2.13" "k8sprow.azurecr.io/kubernetes-e2e-test-images/agnhost:2.13" "k8s.gcr.io/e2e-test-images/agnhost:2.29" "e2eteam/jessie-dnsutils:1.0" "gcr.io/kubernetes-e2e-test-images/jessie-dnsutils:1.0" "e2eteam/pause:3.2" "k8s.gcr.io/pause:3.2" "docker.io/library/busybox:1.29")
+        antrea_images=("sigwindowstools/kube-proxy:v1.18.0" "e2eteam/agnhost:2.13" "us.gcr.io/k8s-artifacts-prod/e2e-test-images/agnhost:2.13" "k8sprow.azurecr.io/kubernetes-e2e-test-images/agnhost:2.13" "registry.k8s.io/e2e-test-images/agnhost:2.29" "e2eteam/jessie-dnsutils:1.0" "gcr.io/kubernetes-e2e-test-images/jessie-dnsutils:1.0" "e2eteam/pause:3.2" "registry.k8s.io/pause:3.2" "docker.io/library/busybox:1.29")
```

Comment:
The antrea_images is used to generate a new tag for the Harbor image to avoid pull rate limit problems. As we still have a Windows Docker testbed with an old version of Kubernetes, the images used in the conformance tests on these testbeds are still in the k8s.gcr.io repository. Perhaps, for deliver_antrea_windows, we can add a new registry.k8s.io tag for e2e tests, but still keep the existing k8s.gcr.io tag for old version conformance tests. What's your opinion? @antoninbas

Reply:
Sorry @XinShuYang, I didn't see your comment before merging this.
I guess we may need to tag the image twice (once with `k8s.gcr.io` and once with `registry.k8s.io`) to keep the tests running?

It feels like there should be a better way to handle this. The current solution seems a bit brittle, as both the Harbor registry and this script need to be updated when a new image is required by the conformance tests.

Also, one thing that comes to mind is that rate limiting doesn't apply to the K8s registries, so why don't we pull them directly?

Reply:
is that why `jenkins-windows-conformance` failed?

Reply:
One of my points is that there should be no rate limiting for `k8s.gcr.io` (unless I am mistaken), so why do we pull these from Harbor?

Reply:
It's possible that images with same version differ on harbor and dockerhub. When I ran "docker pull e2eteam-pause:3.2" on windows node, I received the error message "no matching manifest for windows/amd64 10.0.17763 in the manifest list entries". However, pulling image from "projects.registry.vmware.com/antrea/e2eteam-pause:3.2" worked fine.

Reply:
> Perhaps, for deliver_antrea_windows, we can add a new registry.k8s.io tag for e2e tests, but still keep the existing k8s.gcr.io tag for old version conformance tests. 

@antoninbas @XinShuYang do you recommend to open a new PR for this once we have resolved the best way to tag/pull the image?

Reply:
I will let @XinShuYang decide what is the best course of action here.
Ideally we would not have to list all images like that (or maybe our testbed could be upgraded?).